perses/tests/test_examples.py

```diff
@@ -15,14 +15,14 @@
 import pathlib
 import pytest
 import subprocess
-from perses.tests.utils import enter_temp_directory
+from tempfile import TemporaryDirectory
 
 ROOT_DIR_PATH = pathlib.Path(__file__).joinpath("../../../").resolve()
 
 
 def run_script_file(file_path, cmd_args=None):
     """Run through the shell a python script."""
-    with enter_temp_directory():
+    with TemporaryDirectory():
```

Comment:
I believe the 'undo hacks' commit didn't really work? I think we should try using the `enter_temp_directory` from the perses test utils just to avoid accidental dictionary changes that can happen and to keep things a bit cleaner, since that one removes the tree once it's finished.

Reply:
Sorry I must have missed that! Just pushed the fix